modules/tinymce/src/core/main/ts/table/TableUtils.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+/*
+ NOTE: This file is partially duplicated in the following locations:
+  - models/dom/table/core/TableUtils.ts
+  - plugins/table/core/Utils.ts
+  - advtable
+ Make sure that if making changes to this file, the other files are updated as well
+ */
+
+import { Compare, SugarElement } from '@ephox/sugar';
+
+import Editor from '../api/Editor';
+
+const getBody = (editor: Editor): SugarElement<HTMLElement> =>
```

Comment:
Not something to tackle in this PR, but looking around we use this a fair bit elsewhere. I suspect we could make this a more global utility so might be worth logging a tech debt Jira for that?

Reply:
Have created a Jira (TINY-8484) for us to look into this